core/src/peschke/Complete.scala

```diff
@@ -80,4 +82,12 @@ package peschke
 sealed trait Complete {
```

Comment:
While you're editing this part, can you add `extends Product with Serializable`?